apps/condo/domains/common/constants/import.js

```diff
@@ -1,7 +1,28 @@
+const PROCESSING = 'processing'
+const COMPLETED = 'completed'
+const ERROR = 'error'
+const CANCELLED = 'cancelled'
+
+const IMPORT_STATUS_VALUES = [PROCESSING, COMPLETED, ERROR, CANCELLED]
+
+const DOMA_EXCEL = 'excel'
+const CSV = 'csv'
```

Comment:
These values will be saved to the database. Its better to make names more explicit to prevent migrations in the future.
```suggestion
const DOMA_EXCEL = 'doma-excel'
const CSV = '1s-csv'
```